test/tserialization.nim

```diff
@@ -124,6 +142,19 @@ proc newNode(v: string): ref Node =
 let blockOnly = defineOptions(style=psBlockOnly)
 
 suite "Serialization":
+  test "Internal string assert":
```

Comment:
I get that you tested your code with this, but I'd rather not have tests testing the test code, even less tests that produce random files. Please remove this and the `file` paramater of `assertStringEquals`.

Reply:
Ok, up to you!